apps/web/src/utils/wagmi.ts

```diff
@@ -1,134 +1,57 @@
-import { getWagmiConnector } from '@binance/w3w-wagmi-connector'
-import { CyberWalletConnector, isCyberWallet } from '@cyberlab/cyber-app-sdk'
-import { ChainId } from '@pancakeswap/chains'
-import { BinanceWalletConnector } from '@pancakeswap/wagmi/connectors/binanceWallet'
-import { BloctoConnector } from '@pancakeswap/wagmi/connectors/blocto'
-import { TrustWalletConnector } from '@pancakeswap/wagmi/connectors/trustWallet'
+import { blocto } from '@blocto/wagmi-connector'
 import memoize from 'lodash/memoize'
-import { createConfig, createStorage } from 'wagmi'
-import { CoinbaseWalletConnector } from 'wagmi/connectors/coinbaseWallet'
-import { InjectedConnector } from 'wagmi/connectors/injected'
-import { LedgerConnector } from 'wagmi/connectors/ledger'
-import { MetaMaskConnector } from 'wagmi/connectors/metaMask'
-import { WalletConnectConnector } from 'wagmi/connectors/walletConnect'
-import { chains, publicClient } from './client'
+import { createClient } from 'viem'
+import { createConfig, http } from 'wagmi'
+import { bsc, mainnet } from 'wagmi/chains'
+import { coinbaseWallet, injected, metaMask, walletConnect } from 'wagmi/connectors'
 
-export { chains, publicClient }
+export const chains = [mainnet, bsc]
 
-export const injectedConnector = new InjectedConnector({
-  chains,
-  options: {
-    shimDisconnect: false,
-  },
+export const coinbaseConnector = coinbaseWallet({
+  appName: 'PancakeSwap',
+  appLogoUrl: 'https://pancakeswap.com/logo.png',
 })
 
-export const coinbaseConnector = new CoinbaseWalletConnector({
-  chains,
-  options: {
-    appName: 'PancakeSwap',
-    appLogoUrl: 'https://pancakeswap.com/logo.png',
-  },
+export const walletConnectConnector = walletConnect({
+  projectId: 'e542ff314e26ff34de2d4fba98db70bb',
+  showQrModal: true,
```

Comment:
Are we sure the QR modal is supported in their test environment?

Reply:
e2e tests usually run in a browser headless mode, so i guess some UI behaviors don't work. I'll check if i can run cypress tests and see if everything is green 🙂